### ISSUE TITLE
Problem: Nextcloud cannot find build dir when ENV=dev

### ISSUE THREAD

**Issue by mamedin** [JiscRDSS/rdss-archivematica#221]
When deploying with ENV=dev, I got the error:

```
Step 7/12 : COPY build/files_mv /nextcloud/apps/files_mv
ERROR: Service 'nextcloud' failed to build: COPY failed: stat /var/lib/docker/tmp/docker-builder055534369/build/files_mv: no such file or directory
Makefile:182: recipe for target 'up' failed
make: *** [up] Error 1
```

**Comment by mamedin** [comment]
The `compose/dev/Makefile` needs the same target used on `compose/qa/Makefile` to create this nextcloud `build` directory at Nextcloud source dir:

```
build-nextcloud-apps:
        # Make sure NextCloud apps are built
        @cd $(BASE_DIR)/../../src/rdss-arkivum-nextcloud/ && \
                make build-apps && cd $(BASE_DIR)
```

**Comment by sevein** [comment]
Closed by #222.
